Homework_7.py

```diff
@@ -0,0 +1,95 @@
+# Homework 7
+
+import csv  # import module for work with csv
+import os  # import module for ability to delete files
+from Homework_6 import *  # import previous homeworks (5-6)
+
+
+class CountWords:
+    def parse_file(self, list_with_words):
+        with open('posts.txt', 'r') as source_file:  # open file with posts
+            reader = csv.reader(source_file)  # read each line of the file
```

Comment:
NOT OBLIGATORY TO CHANGE:
I'm not sure if it is necessary to use csv reader for parsing such kind of file. Maybe it is better to use regexp to split file by words.